troubleshooting/troubleshooting.md

```diff
@@ -639,15 +641,56 @@ Issue with AppleRTC, quite a simple fix:
 
 * config.plist -> Kernel -> Quirks -> DisableRtcChecksum -> true
 
-**Note**: If you still have issues, you'll need to use [RTCMemoryFixup](https://github.com/acidanthera/RTCMemoryFixup/releases) and exclude ranges. See [here for more info](https://github.com/acidanthera/bugtracker/issues/788#issuecomment-604608329)
+**Note**: If you still have issues, you'll need to use [RTCMemoryFixup](https://github.com/acidanthera/RTCMemoryFixup/releases) and exclude ranges.
 
-The following boot-arg should handle 99% of cases(pair this with RTCMemoryFixup):
+For a more in-depth guide, see here:
+
+* [Fixing RTC/CMOS Resets](https://dortania.github.io/OpenCore-Post-Install/misc/rtc.html)
+
+## Synaptics PS2 based trackpad doesn't work
+
+You can try to use the [SSDT-Enable_DynamicEWMode.dsl](https://github.com/acidanthera/VoodooPS2/blob/master/Docs/ACPI/SSDT-Enable_DynamicEWMode.dsl).
+First, you have to open DeviceManager, and head to the following:
+
+```
+Device Manager -> Mice and other pointing devices -> Properties -> Details > BIOS device name
+```
+
+Then grab the [SSDT-Enable_DynamicEWMode.dsl](https://github.com/acidanthera/VoodooPS2/blob/master/Docs/ACPI/SSDT-Enable_DynamicEWMode.dsl)
+By default, this uses PCI0.LPCB.PS2K for the pathing. you'll want to rename accordingly.
+
+```
+External (_SB_.PCI0.LPCB.PS2K, DeviceObj) <- Rename this
+
+    Name(_SB.PCI0.LPCB.PS2K.RMCF, Package()  <- Rename this
+
+```
+
+* Note: Although this will work for some case, [the trackpad may be laggy and you can't use the physical buttons](https://github.com/acidanthera/bugtracker/issues/890). If you can't live without the trackpad, this may be better:
+
+You have to find the path of your mouse just like the first solution. We then grab [SSDT-DisableTrackpadProbe.dsl](https://github.com/acidanthera/VoodooPS2/blob/master/Docs/ACPI/SSDT-DisableTrackpadProbe.dsl). By default, this uses PCI0.LPCB.PS2K so you have to rename that to make sure that the SSDT work:
 
 ```
-rtcfx_exclude=00-FF
+External (_SB_.PCI0.LPCB.PS2K, DeviceObj) <- Rename this
+
+    Name(_SB.PCI0.LPCB.PS2K.RMCF, Package() <- Rename this
+```
+
+## Fix for Dell breakless PS2 keyboard keys
```

Comment:
What are `breakless PS2 keyboard keys`?

Reply:
When the key press doesn't release(ie. infinite press), clarified here:
* https://github.com/dortania/OpenCore-Install-Guide/pull/124/commits/8234374410bd67176961541418cf224f16bfb895